conf/config.json

```diff
@@ -40,31 +32,16 @@
                 }
             },
             "destination": {
-                "s3": {
-                    "hosts": [
-                        {
-                            "host": "127.0.0.2",
-                            "port": 9000
-                        }
-                    ],
-                    "transport": "https"
-                },
-                "auth": {
-                    "type": "account",
-                    "account": "lisa",
-                    "vault": {
-                        "hosts": [
-                            {
-                                "host": "127.0.0.2",
-                                "port": 9500
-                            }
-                        ]
-                    }
-                },
+                "transport": "https",
+                "bootstrapList": ["127.0.0.1:8443"],
```

Comment:
I thought about that and realized that for opensource we don't have vault, as long as we pass the destination endpoint as an array this will just work.